src/common/module.c

```diff
@@ -40,7 +40,8 @@ static int run_command(const char *command)
 
 	if (status < 0) {
 		char error_buf[80];
-		strerror_r(errno, error_buf, sizeof(error_buf));
+		char* dummy = strerror_r(errno, error_buf, sizeof(error_buf));
```

Comment:
it seems that freeBSD will compile error. strerror_r returns int in freeBSD。 ref: https://linux.die.net/man/3/strerror_r

is `auto` better?